fastlane/Matchfile

```diff
@@ -1,4 +1,7 @@
-# Store certs/profiles encrypted in Google Cloud
-storage_mode("google_cloud")
-google_cloud_bucket_name("a8c-fastlane-match")
-google_cloud_keys_file(".configure-files/google_cloud_keys.json")
```

Comment:
Sorry for the noise in the diff. While I was on this file, I converted Ruby `""` into `''`, which is the recommended syntax for strings that don't use interpolation.